Automattic/wp-calypso
# FSE: bump newspack-blocks version to 1.5.0

#### Changes proposed in this Pull Request

Bump newspack-blocks version to 1.5.0 to include the [latest changes as listed here](https://github.com/Automattic/newspack-blocks/releases/tag/v1.5.0).

Fixes https://github.com/Automattic/wp-calypso/issues/41934

#### Testing instructions



* Add Posts Carousel block to a page
* See if the 1.5.0 changes are applied e.g. the image is sized `via object-fit:cover`
